openedx_webhooks/lib/github/test/models/test_github_webhook_event.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import arrow
+import pytest
+
+from openedx_webhooks.lib.github.models import GithubWebHookEvent
```

Comment:
I could put test modules alongside the modules they're testing, then we won't need the `test/` subdirectory (which isn't a package, so I literally *can't* do relative imports).

If we had these tests in a project `/tests` directory, we would have to do absolute imports as well, so I guess I don't see the problem.